cid/helpers/athena.py

```diff
@@ -82,10 +84,11 @@ def DatabaseName(self) -> str:
                     self._DatabaseName = default_database.pop().get('Name')
                 else:
                     # Ask user
-                    self._DatabaseName = questionary.select(
-                        "Select AWS Athena database to use",
-                        choices=[d['Name'] for d in athena_databases]
-                    ).ask()
+                    self._DatabaseName = get_parameter(
+                        param_name = 'athena_database',
```

Comment:
Same as above for using dashes instead of underscores

Reply:
yes